polar2grid/compare.py

```diff
@@ -158,7 +174,7 @@ def _get_geotiff_array(gtiff_fn, dtype=None):
     return arr
 
 
-def compare_awips_netcdf(nc1_name, nc2_name, atol=0.0, margin_of_error=0.0, **kwargs) -> list[ArrayComparisonResult]:
+def compare_awips_netcdf(nc1_name, nc2_name, conovert_nan=None, atol=0.0, margin_of_error=0.0, **kwargs) -> list[ArrayComparisonResult]:
```

Comment:
Typo and unused?

Reply:
Oh.  I missed that, this was something I had done before changing to xarray and noticing how the isclose comparison used equal_nan.  I will remove this.

Reply:
Not only that, it was a typo (spelled incorrectly).